src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.TypeNullableDictionary.cs

```diff
@@ -23,11 +24,11 @@ private bool GetIsTypeNullable(Type type)
 
 		private class TypeNullableDictionary
```

Comment:
This dictionary has a single static instance that is kept for the lifetime of the whole app.